lib/datadog/ai_guard/ext.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module Datadog
+  module AIGuard
+    # AI Guard specific constants
+    module Ext
+      SPAN_NAME = "ai_guard"
```

Comment:
I assume the string `ai_guard` is from a specification document correct?
I ask because a span name normally identifies the entity doing the work + the operation being instrumented.
Something like: `ai_guard.evaluate`.